src/persistence/test/MCWSPersistenceProviderSpec.js

```diff
@@ -1,189 +1,186 @@
+            user: {
+                getCurrentUser: () => Promise.resolve({ id: 'myUser' })
+            }
+        };
+
+        spyOn(mcws, 'namespace');
+        mockNamespaceService = jasmine.createSpyObj(
+            'namespaceService',
+            ['getPersistenceNamespaces']
+        );
+        
+        persistenceNamespaces = [
+            {
+                key: 'testSpace',
+                url: '/test/namespace'
+            },
+            {
+                key: 'testSpace2',
+                url: '/test/namespace2'
+            }
+        ];
+
+        mockNamespaceService
+            .getPersistenceNamespaces
+            .and.returnValue(Promise.resolve(persistenceNamespaces));
+
+
+        mockNamespace =
+            jasmine.createSpyObj('namespace', ['opaqueFile', 'read']);
+        mockOpaqueFile = jasmine.createSpyObj('opaqueFile', [
+            'read',
+            'replace',
+            'remove',
+            'create'
+        ]);
+
+        mcws.namespace.and.returnValue(mockNamespace);
+        mockNamespace.opaqueFile.and.returnValue(mockOpaqueFile);
+
+        mcwsPersistenceProvider = new MCWSPersistenceProvider(
+            openmct,
+            persistenceNamespaces
+        );
+    });
 
-        it("provides a listing of identifiers", function (done) {
-            mockNamespace.read.and.returnValue(Promise.resolve([
-                {
-                    Subject: "/test/namespace/a",
-                    Predicate: "has MIO type",
-                    Object: "opaque_file"
-                },
-                {
-                    Subject: "/test/namespace/b",
-                    Predicate: "has MIO type",
-                    Object: "namespace"
-                },
-                {
-                    Subject: "/test/namespace/c",
-                    Predicate: "something else",
-                    Object: "opaque_file"
-                },
-                {
-                    Subject: "/some/namespace/xyz",
-                    Predicate: "has MIO type",
-                    Object: "opaque_file"
-                },
-                {
-                    Subject: "/some/namespace/123-ABC",
-                    Predicate: "has MIO type",
-                    Object: "opaque_file"
-                }
-            ]));
-            persistence
-                .listObjects("testSpace")
-                .then(function (objects) {
-                    expect(objects).toEqual(["a", "xyz", "123-ABC"]);
-                    done();
-                });
-        });
+    it('provides a promise for available namespaces', async (done) => {
+        const spaces = await mcwsPersistenceProvider.getPersistenceNamespaces();
+        expect(spaces).toEqual(persistenceNamespaces);
+        done();
+    });
+    
+    // DO WE DELETE THIS TEST? I don't think we use this functionality anymore.
+    xit('provides a listing of namespaces when provided a namespace definition', async (done) => {
```

Comment:
we do call `getNamespacesFromMCWS` so maybe this is still an okay unit test

Reply:
I'll take a look.